spec/js_routes/options_spec.rb

```diff
@@ -308,12 +308,6 @@
     end
 
     context "when configuring with default_url_options" do
-      context "when default host is not specified" do
-        it "raises an error" do
-          expect { JsRoutes.generate({ :url_links => true }) }.to raise_error RuntimeError
```

Comment:
Note that we no longer raise an error when a default host hasn't been defined because we fallback to `window.location.host` if it has been specified.